CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
 * Updated Certbot and its plugins to improve the handling of file system permissions
   on Windows as a step towards adding proper Windows support to Certbot.
 * Updated urllib3 to 1.24.2 in certbot-auto.
+* Removed the fallback introduced with 0.32.0 in `acme` to retry a response
+  challenge with a `keyAuthorization` after a failed response without this field.
```

Comment:
I'd say:

> ...to retry a challenge response with a `keyAuthorization` if sending the response without this field caused a `malformed` error to be sent by the ACME server.

Also, don't forget to list `acme` in the list of modified packages.

Reply:
Fixed. I juste used "received from" instead of "sent by" to avoid using send two times. Also added acme in the list.